modules/php/Characters/SuzyLafayette.php

```diff
@@ -19,7 +19,7 @@ public function checkHand()
     if ($this->getHand()->count() == 0 && Rules::isAbilityAvailable()) {
       $ctx = Stack::getCtx();
       $newAtom = Stack::newSimpleAtom(ST_TRIGGER_ABILITY, $this);
-      if ($ctx['state'] === ST_REACT && $ctx['missedNeeded'] > 1) {
+      if ($ctx['state'] === ST_REACT && array_key_exists('missedNeeded', $ctx) && $ctx['missedNeeded'] > 1) {
```

Comment:
Usually isset() is used to do this kind of checks. array_key_exists() should be used only if you are interested in null values since isset() will return false if array_key_exists() but it's value is null.

Reply:
Ok, thanks, will know that :)